README.md

```diff
@@ -81,3 +81,8 @@ Notes on Test Coverage
 ------------------
 
 You can see your [test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) when you run your tests.
```

Comment:
Have you considered adding some, install/usage guide? Would be great to have a little irb run through